### DISCUSSION TITLE
Sort repositories alphabetically

### DISCUSSION THREAD

**Discussion by mdwhatcott** [community/community#137890] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.176.0 (140321802)
Device: iPhone SE (2nd generation) (iOS 17.6.1)
</details>

**Comment by rahulkottikasala** [comment]
Got it! Let me know if you need any help related to this version of GitHub or if there's a specific issue you'd like to debug on your iPhone SE with iOS 17.6.1.
